### DISCUSSION TITLE
What audio filter(s) do you put in your mpv config

### DISCUSSION THREAD

**Discussion by cmdrkotori** [mpv-player/mpv#13945] [General]
# Preamble

It's come to the point where some users are asking for audio filters for my frontend, mpc-qt.  But ffmpeg comes with ~300 filters, so supporting all of them in gui is practically impossible; I will need to be very picky about what filters I expose and spend development time on if it is to have any chance of success.

I need some feedback on what a good/expected set of mpv audio filters looks like.  As I have a good pair of headphones, (Sennheiser HD 558,) I have never felt the need to use any audio filters while using mpv.

# What I'd like to know

What audio filters do you use?  What order do you put your filters in?

Cheers thanks.

**Comment by geextahslex** [comment]
Hi, what do you mean by "audio filters"? can you give some examples?
I personally use VST plugins on my machine to shape audio but this is outisde of mpv.

**Comment by cmdrkotori** [comment]
Example: `mpv --af=lavfi-loudnorm`

You can see the list by running `mpv --af=help`

**Comment by richardpl** [comment]
loudnorm filter (from FFmpeg project), especially dynamic default single-pass mode is low quality, dunno why people use it all the time.

**Reply by hooke007** [reply]
> dunno why people use it all the time.

Most people doesn't care much about quality, but they even cannot get any useful built-in alternative from mpv.
